setup.py

```diff
@@ -10,6 +10,7 @@
 import sys
 import runpy
 from setuptools import setup, find_packages
+from pull_demographic_data import load_sources
```

Comment:
This works when you do pip install synthpops but not when you do "python setup.py develop". It results in:
```
Traceback (most recent call last):
  File "C:\git\covasim\clean\covasim\covasim\requirements.py", line 20, in check_sciris
    import sciris as sc
ModuleNotFoundError: No module named 'sciris'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 13, in <module>
    from pull_demographic_data import load_sources
  File "C:\git\covasim\clean\covasim\pull_demographic_data.py", line 13, in <module>
    from covasim.datasets.data_loader import load_country_pop
  File "C:\git\covasim\clean\covasim\covasim\__init__.py", line 6, in <module>
    from . import requirements
  File "C:\git\covasim\clean\covasim\covasim\requirements.py", line 92, in <module>
    check_sciris()
  File "C:\git\covasim\clean\covasim\covasim\requirements.py", line 23, in check_sciris
    raise ModuleNotFoundError(errormsg)
ModuleNotFoundError: Sciris is a required dependency but is not found; please install via "pip install sciris"
```

Reply:
We definitely want this as a script the user can run, not something to run on setup

Reply:
That failure is saying it cannot install `Sciris` and is not related to the work in this pr.

Reply:
I can take it out of the setup, I wasn't sure if we would want it there or not, so I just added it since it seemed harmless and related to setup.